widgetsnbextension/src/extension.js

```diff
@@ -46,7 +50,7 @@ var handle_cell = function(cell) {
     }
 };
 
-function register_events(Jupyter, events) {
+function register_events(Jupyter, events, outputarea) {
```

Comment:
Yeah, there are other kludgy workarounds I can now delete because we don't have webpack too, like https://github.com/jupyter-widgets/ipywidgets/blob/master/widgetsnbextension/src/widget_output.js#L93